changelog_unreleased/blog-post-intro.md

```diff
@@ -0,0 +1,17 @@
+---
+author: "Sosuke Suzuki (@sosukesuzuki)"
+authorURL: "https://github.com/sosukesuzuki"
+title: "Prettier 3.0: Hello, ECMAScript Modules!"
+---
+
+We are excited to announce the release of the new version of Prettier!
+
+We have made the migration to using ECMAScript Modules for all our source code. This change has significantly improved the development experience for the Prettier team. Please rest assured that when using Prettier as a library, you can still use it as CommonJS as well.
+
+This update comes with several breaking changes. One notable example is the alteration in markdown formatting - spaces are no longer inserted between Latin characters and Chinese or Japanese characters. We'd like to extend our gratitude to [Tatsunori Uchino](https://github.com/tats-u), who has made significant contributions to Prettier over the past year, particularly with this feature. Additionally, the default value of `trailingComma` has been changed to `"all"`.
```

Comment:
@tats-u  We greatly appreciate your contribution in improving the formatting of Markdown. So we would like to put your name at the top of the Prettier v3 release blog, is that okay?

Reply:
@sosukesuzuki Sorry for very late reply. There's no problem to post my name.